build.gradle

```diff
@@ -117,6 +118,7 @@ dependencies {
     runtimeOnly "commons-beanutils:commons-beanutils:${versions.beanUtils}"
     runtimeOnly "mysql:mysql-connector-java:${versions.mysqlConnector}"
 
+    // TODO not sure of the scope here. i suspect we might extend some jupiter classes
```

Comment:
Is this still necessary?

Reply:
i can remove it, i would like to avoid extending jupiter engine if possible